src/util/backtrace.cc

```diff
@@ -41,7 +41,7 @@ static int dl_iterate_phdr_callback(struct dl_phdr_info *info, size_t size, void
         const auto hdr = info->dlpi_phdr[i];
 
         // Only account loadable, executable (text) segments
-        if (hdr.p_type == PT_LOAD && (hdr.p_flags & PF_X) == PF_X) {
+        if (hdr.p_type == PT_LOAD) {
```

Comment:
Comment is now incorrect.

Reply:
Well it's now incorrect in a different way.

Reply:
Huh, I thought I corrected the comment.
Good catch, fixed.